app/views/admin/paypal_preferences/index.haml

```diff
@@ -60,13 +54,27 @@
                 %h3.paypal-wizard-step
                   - if paypal_prefs_valid
                     =icon_tag("check", ["icon-fix", "paypal-success-mark"])
-                  =t("admin.paypal_accounts.set_minimum_price_and_fee")
+                  =t("admin.paypal_accounts.edit_payment_settings")
+            .row
+              .col-12
+                %p.paypal-currency-information
+                  =icon_tag("information")
+                  =t("admin.paypal_accounts.supported_currencies_information_text",
+                     contact_support_link: link_to(t("admin.paypal_accounts.contact_support_link_text"),
+                                                   "mailto:support@sharetribe.com")).html_safe
```

Comment:
We shouldn't hard-code the support email here. How have we done these suppot links in other parts of the app? In addition to that, should we use the `data-uv-trigger` tag to use the UserVoice widget?